src/routes/_header.svelte

```diff
@@ -15,6 +15,10 @@
 
 <script lang="ts">
   import { namespace } from '$lib/stores/namespace';
+  import { StatusOnline, StatusOffline } from 'svelte-hero-icons';
```

Comment:
This is totally cool for now, but let's make a mental note that Candace has new icons coming.

Reply:
Yup! I've talked to Candace about the how this works and the icons I'm using. Also about the future state of how this will be decrypted potentially on a per field basis, instead of the every field at once thing we have going on now.